schema/me/conversations.js

```diff
@@ -1,21 +1,35 @@
 import { pageable } from "relay-cursor-paging"
 import { parseRelayOptions } from "lib/helpers"
+import { GraphQLInt } from "graphql"
 import { connectionFromArraySlice, connectionDefinitions } from "graphql-relay"
+import { assign } from "lodash"
 
 import { ConversationType } from "./conversation"
 
 export default {
-  type: connectionDefinitions({ nodeType: ConversationType }).connectionType,
+  type: connectionDefinitions({
+    nodeType: ConversationType,
+    connectionFields: {
+      totalUnreadCount: {
+        type: GraphQLInt,
+        resolve: ({ total_unread_count }) => total_unread_count,
+      },
+    },
+  }).connectionType,
   description: "Conversations, usually between a user and partner.",
   args: pageable(),
   resolve: (root, options, request, { rootValue: { conversationsLoader } }) => {
     if (!conversationsLoader) return null
     const { page, size, offset } = parseRelayOptions(options)
-    return conversationsLoader({ page, size }).then(({ total_count, conversations }) => {
-      return connectionFromArraySlice(conversations, options, {
-        arrayLength: total_count,
-        sliceStart: offset,
-      })
+    const expand = ["total_unread_count"]
+    return conversationsLoader({ page, size, expand }).then(({ total_count, total_unread_count, conversations }) => {
+      return assign(
+        { total_unread_count },
```

Comment:
Here's where we merge in the `total_unread_count` into the connection. The spec shows the format:

```
conversations {
  pageInfo {
    ...
  }
  totalUnreadCount
  edges {
    node {
      ....
```

Reply:
Also went for 🐫 this time!